aptos-move/framework/aptos-framework/sources/configs/Stake.move

```diff
@@ -456,9 +457,33 @@ module AptosFramework::Stake {
             };
             i = i + 1;
         };
+
+        sort_validators(&mut active_validators);
         validator_set.active_validators = active_validators;
     }
 
+    fun sort_validators(validators: &mut vector<ValidatorInfo>) {
+        let length = Vector::length(validators);
+        if (length == 0) {
+            return
+        };
+
+        let ordered = false;
+        while (!ordered) {
```

Comment:
What is this? Bubble sort? 😵‍💫 Can this tank performance here if the validator set is large enough (thousands)?

Reply:
Maybe, but remember this is used once a day or a few times a day. This is something that can easily be worked on in the background. Other searching schemes involve more complexity and testing -- especially in Move. I'm not keen to do that given the constraints -- the lack of callbacks / closures.

Bubble sort ftw.

Reply:
That makes sense! I think this is a new achievement unlocked though - implementing bubble sort on-chain 😆